src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -661,7 +661,7 @@ bool emberAfOperationalCredentialsClusterAttestationRequestCallback(app::Command
     Platform::ScopedMemoryBuffer<uint8_t> attestationElements;
     size_t attestationElementsLen = 0;
     MutableByteSpan attestationElementsSpan;
-    uint8_t certDeclBuf[512];
+    uint8_t certDeclBuf[541]; // Sized to hold the example certificate declaration wtih 100 PIDs. See DeviceAttestationCredsExample
```

Comment:
```suggestion
    uint8_t certDeclBuf[kMaxCMSSignedCDMessage]; // Sized to hold the example certificate declaration wtih 100 PIDs. See DeviceAttestationCredsExample
```

Reply:
Where is that defined? What is the max CD size?

Reply:
It's in `src/credentials/CertificationDeclaration.h`, looks like?

Reply:
cool thanks. Added the header and changed this in the next commit.